website/views.py

```diff
@@ -166,6 +166,11 @@ def resolve_guid(guid, suffix=None):
     guid_object = Guid.load(guid)
     if guid_object:
 
+        # redirect if user tries to visit public files wiki or add contributors ect.
+        if Node.find(Q('_id', 'eq', guid_object._storage_key) & Q('is_public_files_collection', 'eq', True)):
```

Comment:
I'm not certain that this should be happening here, or at all. This is intended to prevent a user from seeing the error (raised by `disable_for_public_files_collection`) that happens as a result of manually navigating to a route that is 1) disabled for public files and 2) hidden from the public files page UI, correct? If that's the case, it seems unnecessary to redirect them -- it should just `404`.

Also, you shouldn't use `_storage_key`; `_id` is preferred.